### PR TITLE
feat: Add persistent store

### PR DESCRIPTION
**Pull request by GeckoEidechse**
Adds persistent store using [`tauri-plugin-store`](https://github.com/tauri-apps/tauri-plugin-store).

For now, it's only used to store the selected release canal.

### CODE REVIEW

File: src-vue/package.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
   "dependencies": {
     "@element-plus/icons-vue": "^2.0.9",
     "element-plus": "^2.2.17",
+    "tauri-plugin-store-api": "github:tauri-apps/tauri-plugin-store#dev",
```
Comment (Alystrasz) [inline src-vue/package.json:14]: The #dev means npm should use `dev` branch?

**Comment by Alystrasz** [comment]
Also, I think we need a readme part where we explain difference between different application stores

File: src-vue/package.json (hunk 1 above)
Comment (GeckoEidechse) [reply src-vue/package.json:14]: Yup, there's no tagged release yet unfortunately: https://github.com/tauri-apps/tauri-plugin-store

I basically just did `npm install https://github.com/tauri-apps/tauri-plugin-store\#dev`

**Comment by GeckoEidechse** [comment]
> Also, I think we need a readme part where we explain difference between different application stores

Where should we put it? Add it to the one in `src-vue`? Cause the readme in repo root is too end-user facing IMO ^^

File: src-vue/package.json (hunk 1 above)
Comment (Alystrasz) [reply src-vue/package.json:14]: Aren't there more stable branches to rely on?

**Comment by Alystrasz** [comment]
Well, GitHub is a development platform after all :)
We could put it under the `Development` section.

**Pull request merged**